ipatests/prci_definitions/nightly_latest_testing_selinux.yaml

```diff
@@ -1729,8 +1847,9 @@ jobs:
 #        build_url: '{testing-fedora/build_url}'
 #        update_packages: True
 #        selinux_enforcing: True
+enable_testing_repo: True
```

Comment:
This line should be commented

Reply:
and it will fix lint error:
Your YAML file: ./ipatests/prci_definitions/nightly_latest_testing_selinux.yaml has a wrong syntax or this is a Jinja template. In the latter clause, consider to add your YAML file to the YAML_TEMPLATE_FILES list in Makefile.am.
make: *** [Makefile:1344: yamllint] Error 1

Reply:
Thanks!